usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -106,7 +120,125 @@ def prepare_spark(self):
         if spark_created_by_command:
             self.spark.stop()
 
-    def load_transaction_lookup_ids(self, last_etl_date):
+    def delete_records_sql(self):
+        if self.etl_level == "transaction_id_lookup":
+            id_col = "id"
+            subquery = """
+                SELECT id AS id_to_remove
+                FROM int.transaction_id_lookup AS tidlu LEFT JOIN raw.detached_award_procurement AS dap ON (
+                    tidlu.detached_award_procurement_id = dap.detached_award_procurement_id
+                )
+                WHERE tidlu.detached_award_procurement_id IS NOT NULL AND dap.detached_award_procurement_id IS NULL
+                UNION ALL
+                SELECT id AS id_to_remove
+                FROM int.transaction_id_lookup AS tidlu LEFT JOIN raw.published_fabs AS pfabs ON (
+                    tidlu.published_fabs_id = pfabs.published_fabs_id
+                )
+                WHERE tidlu.published_fabs_id IS NOT NULL AND pfabs.published_fabs_id IS NULL
+            """
+        elif self.etl_level == "award_id_lookup":
+            id_col = "transaction_unique_id"
+            subquery = """
+                SELECT aidlu.transaction_unique_id AS id_to_remove
+                FROM int.award_id_lookup AS aidlu LEFT JOIN raw.detached_award_procurement AS dap ON (
+                    aidlu.detached_award_procurement_id = dap.detached_award_procurement_id
+                )
+                WHERE aidlu.detached_award_procurement_id IS NOT NULL AND dap.detached_award_procurement_id IS NULL
+                UNION ALL
+                SELECT aidlu.transaction_unique_id AS id_to_remove
+                FROM int.award_id_lookup AS aidlu LEFT JOIN raw.published_fabs AS pfabs ON (
+                    aidlu.published_fabs_id = pfabs.published_fabs_id
+                )
+                WHERE aidlu.published_fabs_id IS NOT NULL AND pfabs.published_fabs_id IS NULL
+            """
+        else:
```

Comment:
I'd still call out specific `etl_level`s here.

Reply:
Why?  That would result in code duplication.

Reply:
Mostly for readability, so the reader doesn't have to cross reference the list of possible `etl_levels` to figure out what scenarios will cause this code to be activated and how it will behave.
In order to avoid code duplication, you can do something like this: 
```
elif self.etl_level in ("transaction_fpds", "transaction_fabs")
```

Reply:
ok